tests/testsuite/bad_config.rs

```diff
@@ -2151,6 +2151,37 @@ Caused by:
         .run();
 }
 
+#[cargo_test]
+fn github_pull_request_url() {
```

Comment:
As noted in #14941, this should cover patches as well and we should have a test for it.

Reply:
#15001 you mean?

Reply:
Yes :)

Reply:
Done in d743987ca